report/src/problem_analysis/hci/prototype_evaluation.tex

```diff
@@ -1,3 +1,71 @@
+
+Expert-based usability testing is often carried out as an inspection method \cite{NJ_InspectionMethods} by interface experts that have not participated in the creation of the design.
+This method utilizes the experts' knowledge of interface design to quickly assess and correct flaws in the design.
+Their lack of knowledge about the tasks to perform is used to discover flaws in the way they interact with the system \cite{researchmethodsinhumancomputerinteraction}.
+However, these types of tests are most useful when performed before a test involving users\cite{lazar2005web}. 
+This means that the test effort becomes larger, as both expert and user tests should be performed to optimize the number of found issues. \cite{researchmethodsinhumancomputerinteraction} 
+
+Many types of inspection methods can be used when performing expert-based usability testing.
+Consistency inspection involves establishing whether the design is consistent with existing designs \cite{NJ_InspectionMethods,wixon_consistencyInspections}.  
+Cognitive walkthroughs involve experts acting as if they were system users performing a series of tasks with the prototype \cite{Hollingsed_cognitive_walthrough}.
+
+A different approach to evaluation is user-based testing.
+User-based testing is often performed using a group of users representative of the overall population of users. \cite{researchmethodsinhumancomputerinteraction}.
+User-based testing is based on the idea that designers have preconceived opinions about how a system should be interacted with. 
+Therefore, the designers are likely to create designs that suit their view of how a design should be.However, this design does not necessarily align with how users imagine the system \cite{user-centred-design,researchmethodsinhumancomputerinteraction}.
+Two approaches can be used when conducting user-based testing. 
+Formative user-based testing is used early in the design phase to evaluate early designs. 
+The evaluation mainly focuses on finding design alternatives and often uses low-fidelity prototypes \cite{dumasAndFox_formative_usability_testing}.
+
+Contrary to formative testing, summative testing is performed late in the development of the design and often uses a hifi prototype.
+Here, the focus is on the evaluate whether the design satisfies specific design goals. \cite{dumasAndFox_formative_usability_testing}
+In our case, the design goals would constitute whether the design solves the problems described in Section \ref{sec:understanding_the_problem}..
+
+When choosing which type of usability test to use, we consider its strengths in terms of whether resources for the test approach can be realistically allocated by all parties and whether the approach solves the problematic situations.
```

Comment:
Almost complete duplicate sentence of line 8 ??

Reply:
Good catch, we yeet this line and keep line 8